docs/devs/followerNode_from_genesis.md

```diff
@@ -0,0 +1,239 @@
+
+Example:
+
+```bash
+
+ansible-playbook --inventory ec2-18-144-5-233.us-west-1.compute.amazonaws.com, \
+    --user rahat  \
+    --extra-vars "movement_container_version=6e00b778ee7d8139b153aa4eb80805aead07e252" \
+    --extra-vars "user=rahat" \
+    docs/movement-node/run/ansible/follower-node/mainnet/movement-full-follower.yml \
+    --private-key rahat_deployment_test.pem
+
+```
+
+#### Bardock Testnet
+
+For Bardock Testnet, please use the following example:
+
+```bash
+
+ansible-playbook --inventory ec2-18-144-5-233.us-west-1.compute.amazonaws.com, \
+    --user rahat  \
+    --extra-vars "movement_container_version=6e00b778ee7d8139b153aa4eb80805aead07e252" \
+    --extra-vars "user=rahat" \
+    docs/movement-node/run/ansible/follower-node/movement-full-follower.yml \
+    --private-key rahat_deployment_test.pem
+
+```
+
+#### After installation
+
+If the service doesn't start, often it's because the config.json file hasn't been created correctly.
+
+To solve the issue replace the provided [config.json](config.json) in the $HOME/.movement folder and start the service with the command:
+
+```bash
+systemctl start movement-full-follower.service 
+
+```
+
+### Verify Deployment
+
+- Connect to your instance (in the above example I ssh into my instance)
+- run the command **`curl localhost:30731/v1`**
+- Output should be similar to this:
+
+```json
+
+{"chain_id":126,"epoch":"1","ledger_version":"9","oldest_ledger_version":"0","ledger_timestamp":"1732636319660843","node_role":"validator","oldest_block_height":"0","block_height":"3","git_hash":"9dfc8e7a3d622597dfd81cc4ba480a5377f87a"}
+
+```
+
+Compare to: 
+
+- Mainnet endpoint: https://mainnet.movementnetwork.xyz/v1
+- Testnet endpoint: https://aptos.testnet.bardock.movementlabs.xyz/v1
+
+## Run Locally 
+
+It is easiest to gain quick familiarity with the Movement Follower Node by running it locally.
+
+To do this, ensure you have nix installed. We recommend the Determinate Systems nix installation script. You can find it [here](https://determinate.systems/posts/determinate-nix-installer/).
+
+```bash
+
+curl --proto '=https' --tlsv1.2 -sSf -L https://install.determinate.systems/nix | sh -s -- install
+
+```
+
+After installing `nix`, clone the Movement repository and open the `nix-shell` environment.
+
+```bash
+
+# if you have not cloned the repository yet
+git clone https://github.com/movementlabsxyz/movement
+cd movement
+git checkout 6e00b778ee7d8139b153aa4eb80805aead07e252
+nix develop
+
+```
+
+This should install all dependencies needed to work on the Movement Follower Node.
+
+You can now either run the follower node natively or with our containers via the provided just commands.
+
+First create, an environment file for the follower node. The example below is for the Movement Testnet Bardock. Comments are made on how to change the environment file for other networks.
+
+```
+CONTAINER_REV=6e00b778ee7d8139b153aa4eb80805aead07e252
+DOT_MOVEMENT_PATH=./.movement
```

Comment:
Is this correct `./.movement`? Should it not be `.movement/`?

Reply:
It's the same. I take this part from the original doc so I didn't change it.